src/library.coffee

```diff
@@ -104,19 +104,28 @@ class Library extends EventEmitter
 
     @components = {} # "name" -> { command: "", language: ''|null }.  lazy-loaded using load()
 
+  getComponent: (name) ->
+    # Direct match
+    return @components[name] if @components[name]
+    withoutNamespace = path.basename name
```

Comment:
Would love to, but at least the current [doc on setting up foreign participants](https://github.com/msgflo/msgflo#using-non-msgflo-aware-code-as-participants) is non-namespaced, so there might be these in the wild